### PR TITLE
SK-1487-WEB-Emails-all_offers-manage_password

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password/guide.fr-fr.md [hunk 1]
```diff
@@ -100,7 +100,15 @@ Sur la nouvelle fenêtre qui apparaît, commencez par renseigner votre mot de pa
 
 ![email](images/mxplan-password-new-step5.png){.thumbnail}
 
-#### Identifiez votre offre MXplan <a name="whichmxplan"></a>
+### Récupérer un mot de passe
+
+Pour des raisons de sécurité et de confidentialité, il n'est pas possible de **récupérer** un mot de passe. Comme il est décrit dans les étapes précédentes, il est nécessaire de le réinitialiser si vous ne le connaissez pas.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password/guide.fr-fr.md]:
```suggestion
Pour des raisons de sécurité et de confidentialité, il n'est pas possible de **récupérer** un mot de passe. Comme cela est décrit dans les étapes précédentes, il est nécessaire de réinitialiser le mot de passe si vous ne le connaissez plus.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password/guide.fr-fr.md [hunk 2]
```diff
@@ -100,7 +100,15 @@ Sur la nouvelle fenêtre qui apparaît, commencez par renseigner votre mot de pa
...
+
+> [!primary]
+>
+> Si vous souhaitez stocker un mot de passe, il est conseillé d'utiliser un gestionnaire de mot de passe.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password/guide.fr-fr.md]:
```suggestion
> Si vous souhaitez stocker un mot de passe, nous vous conseillons d'utiliser un gestionnaire de mot de passe, comme Keepass par exemple.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-emails/guide.fr-fr.md [hunk 3]
```diff
@@ -55,6 +55,14 @@ Le **mot de passe** doit respecter les règles suivantes :
 > [!warning]
 > Pour des raisons de sécurité, n'utilisez pas deux fois le même mot de passe. Choisissez-en un qui n'a aucun rapport avec vos informations personnelles (évitez par exemple de mentionner vos nom, prénom et date de naissance). Changez-le régulièrement.
 
+### Comment récupérer mon mot de passe oublié ?
+
+Pour des raisons de sécurité et de confidentialité, il n'est pas possible de **récupérer** un mot de passe. Comme il est décrit dans notre guide "[Modifier le mot de passe d'une adresse e-mail](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password)", il est nécessaire de le réinitialiser si vous ne le connaissez pas.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-emails/guide.fr-fr.md]:
```suggestion
Pour des raisons de sécurité et de confidentialité, il n'est pas possible de **récupérer** un mot de passe. Comme cela est décrit dans notre guide « [Modifier le mot de passe d'une adresse e-mail](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password) », il est nécessaire de réinitialiser votre mot de passe si vous ne le connaissez plus.
```

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-emails/guide.fr-fr.md [hunk 4]
```diff
@@ -55,6 +55,14 @@ Le **mot de passe** doit respecter les règles suivantes :
...
+
+> [!primary]
+>
+> Si vous souhaitez stocker un mot de passe, il est conseillé d'utiliser un gestionnaire de mot de passe.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-emails/guide.fr-fr.md]:
```suggestion
> Si vous souhaitez stocker un mot de passe, nous vous conseillons d'utiliser un gestionnaire de mot de passe, comme Keepass par exemple.
```

**Comment by BrenKt** [comment]
Translations to check

File: pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-emails/guide.pt-pt.md [hunk 5]
```diff
@@ -59,6 +59,14 @@ A **palavra-passe** deve respeitar as seguintes regras:
 > [!warning]
 > Por razões de segurança, recomendamos que não utilize duas vezes a mesma palavra-passe. Escolha uma palavra-passe que não possa ser relacionada com as suas informações pessoais (evite qualquer referência ao seu sobrenome, nome ou data de nascimento, por exemplo) e renove-a regularmente.
 
+### Como recuperar a minha palavra-passe esquecida?
+
+Por razões de segurança e de confidencialidade, não é possível **recuperar** uma palavra-passe. Conforme descrito no nosso "[Alterar a palavra-passe de um endereço de e-mail](/pages/web_cloud/email_and_colaborative_solutions/mx_plan/email_change_password)", é necessário redefinir a sua palavra-passe se não a souber mais.
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/mx_plan/faq-emails/guide.pt-pt.md]:
```suggestion
Por razões de segurança e de confidencialidade, não é possível **recuperar** uma palavra-passe. Conforme descrito no nosso "[Alterar a palavra-passe de um endereço de e-mail](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_change_password)", é necessário redefinir a sua palavra-passe se não a souber mais.
```

**Review by Kazarus4 — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**